feedback-module/src/components/common/ErrorAlert.js

```diff
@@ -2,17 +2,17 @@ import React from "react";
 import { Alert } from "@trussworks/react-uswds";
 import FeatherIcon from "feather-icons-react";
 
-function ErrorAlert({ errorText, dir, id }) {
+function ErrorAlert({ errorText, id, className }) {
   return (
-    <Alert type="error" slim noIcon wrapper="span" id={id}>
-      <FeatherIcon
-        icon="alert-triangle"
-        size="22"
-        strokeWidth="2.5"
-        className={`${
-          dir === "rtl" ? "margin-left-205" : "margin-right-205"
-        } padding-top-2px`}
-      />
+    <Alert
+      type="error"
+      slim
+      noIcon
+      wrapper="span"
+      id={id}
+      className={className ?? ""}
```

Comment:
I think you an just set this as `className={className}`

Reply:
It compiles as class="undefined" in the html if we don't actually provide a className, so I'm not sure if it's best to keep it like that.

Reply:
Hmm ok. I don't know if we want it to read as className="" either though

Reply:
Found a solution! Looks like setting it to `null` instead works